README.md

```diff
@@ -1,3 +1,82 @@
-# Instant CLIP Tokenizer: a fast tokenizer for the CLIP neural network
+![Cover logo](./cover.svg)
 
-The vocabulary file and Python tokenizer code in this repository are Copyright (c) 2021 OpenAI ([MIT-License](https://github.com/openai/CLIP/blob/main/LICENSE)).
+# Instant CLIP Tokenizer: a fast tokenizer for the CLIP neural network, written in Rust
```

Comment:
Maybe leave out the "written in Rust" here? It feels tacked on and isn't super important to the message IMO.